tools/candid-wasm-bindgen/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "candid-wasm-bindgen"
+version = "0.1.0"
+authors = ["Kyle Peacock <kylpeacock@gmail.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["console_error_panic_hook"]
+
+[dependencies]
+wasm-bindgen = "0.2.63"
+candid = { path = "../../rust/candid" }
+
+# The `console_error_panic_hook` crate provides better debugging of panics by
+# logging them with `console.error`. This is great for development, but requires
+# all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
+# code size when deploying.
+console_error_panic_hook = { version = "0.1.6", optional = true }
+
+# `wee_alloc` is a tiny allocator for wasm that is only ~1K in code size
+# compared to the default allocator's ~10K. It is slower than the default
+# allocator, however.
+wee_alloc = { version = "0.4.5", optional = true }
```

Comment:
I think we stayed away from wee_alloc, because it's too aggressive in asking for new memory pages: https://dfinity.slack.com/archives/CH4CADCJX/p1615932677024900. Do things improve since then?

Reply:
Relevant conversation: https://github.com/rustwasm/wee_alloc/issues/107

Likewise, here's a security alert I received a few weeks ago for `prettier-plugin-motoko`:

![image](https://user-images.githubusercontent.com/522097/193710717-9094fc8c-8215-4688-a0d8-74990f33bd08.png)

It looks like `wee_alloc` is disabled by default, so perhaps the main reason to remove the dependency is to avoid a similar Dependabot message. Should be fine either way.

Reply:
Nice! We observed the same memory leak a year ago, and was assuming something was wrong on our platform. Because we didn't see any bug reports about it with such a popular crate...